app/middlewares/users.js

```diff
@@ -1,19 +1,29 @@
-const { body } = require('express-validator');
+const { checkSchema } = require('express-validator');
 
 const { fieldsValidation } = require('./fields_validation');
 const { findUserByEmail } = require('../services/users');
 const { userEmailRepeatedError } = require('../errors');
 
-exports.userBodyValidations = fieldsValidation([
-  body('first_name', 'first_name must be present')
-    .not()
-    .isEmpty(),
-  body('last_name', 'last_name must be present')
-    .not()
-    .isEmpty(),
-  body('password', 'password must have at least 7 characters').isLength({ min: 7 }),
-  body('email', 'email must have email format').isEmail()
-]);
+exports.userBodyValidations = fieldsValidation(
```

Comment:
I would remove this from users validation, and call the `fields_validations` function from the routes. Something like:
`app.post(..., [fieldsValidation(schema), ...], ...)`